packages/react/src/types/component-props.ts

```diff
@@ -265,141 +317,168 @@ export interface ReactFlowProps<NodeType extends Node = Node, EdgeType extends E
    * The type of edge path to use for connection lines.
    *
    * Although created edges can be of any type, React Flow needs to know what type of path to render for the connection line before the edge is created!
+   * @default ConnectionLineType.Bezier
    */
   connectionLineType?: ConnectionLineType;
-  /** Styles to be applied to the connection line */
+  /** Styles to be applied to the connection line. */
   connectionLineStyle?: CSSProperties;
-  /** React Component to be used as a connection line */
+  /** React Component to be used as a connection line. */
   connectionLineComponent?: ConnectionLineComponent<NodeType>;
-  /** Styles to be applied to the container of the connection line */
+  /** Styles to be applied to the container of the connection line. */
   connectionLineContainerStyle?: CSSProperties;
   /**
-   * 'strict' connection mode will only allow you to connect source handles to target handles.
-   *
-   * 'loose' connection mode will allow you to connect handles of any type to one another.
+   * A loose connection mode will allow you to connect handles with differing types, including
+   * source-to-source connections. However, it does not support target-to-target connections. Strict
+   * mode allows only connections between source handles and target handles.
    * @default 'strict'
    */
   connectionMode?: ConnectionMode;
   /**
-   * Pressing down this key deletes all selected nodes & edges.
+   * If set, pressing the key or chord will delete any selected nodes and edges. Passing an array
+   * represents multiple keys that can be pressed.
+   *
+   * For example, `["Delete", "Backspace"]` will delete selected elements when either key is pressed.
    * @default 'Backspace'
    */
   deleteKeyCode?: KeyCode | null;
   /**
-   * If a key is set, you can pan the viewport while that key is held down even if panOnScroll is set to false.
+   * If set, holding this key will let you click and drag to draw a selection box around multiple
+   * nodes and edges. Passing an array represents multiple keys that can be pressed.
    *
-   * By setting this prop to null you can disable this functionality.
-   * @default 'Space'
+   * For example, `["Shift", "Meta"]` will allow you to draw a selection box when either key is
+   * pressed.
+   * @default 'Shift'
```

Comment:
Here it was mistake 😄https://github.com/xyflow/xyflow/blob/b9e4f829339c2e3dd489dba2f207e11fb7c67f7c/packages/react/src/container/ReactFlow/index.tsx#L69